1001-2000/LIT1340EnochE.xml

```diff
@@ -48,30 +50,120 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+               <witness corresp="BNFabb16">BnF Éthiopien d'Abbadie 16</witness>
+               <witness corresp="BNFabb30">BnF Éthiopien d'Abbadie 30</witness>
+               <witness corresp="BNFabb35">BnF Éthiopien d'Abbadie 35</witness>
+               <witness corresp="BNFabb55">BnF Éthiopien d'Abbadie 55</witness>
+               <witness corresp="BNFabb99">BnF Éthiopien d'Abbadie 99</witness>
+               <witness corresp="BNFabb197">BnF Éthiopien d'Abbadie 197</witness>
+               <witness corresp="BNFet49">BnF Éthiopien 49</witness>
+               <witness corresp="BNFet50">BnF Éthiopien 50</witness>
+               <witness corresp="BDLbruce74">Bodleian Bruce 74</witness>
+               <witness corresp="BDLor531">Bodleian Orient 531</witness>
+               <witness corresp="BLadd24185">BL Additional 24185</witness>
+               <witness corresp="BLadd24990">BL Additional 24990</witness>
+               <witness corresp="BLorient484">BL Oriental 484</witness>
+               <witness corresp="BLorient485">BL Oriental 485</witness>
+               <witness corresp="BLorient490">BL Oriental 490</witness>
+               <witness corresp="BLorient491">BL Oriental 491</witness>
+               <witness corresp="BLorient492">BL Oriental 492</witness>
+               <witness corresp="BLorient499">BL Oriental 499</witness>
+               <witness corresp="BLorient8822">BL Oriental 8822</witness>
+               <witness corresp="BLorient8823">BL Oriental 8823</witness>
+               <witness corresp="CamAdd1570" xml:id="camadd1570">CUL Add. 1570</witness>
+               <witness corresp="EMML36" xml:id="emml36">EMML 36</witness>
+               <witness corresp="EMML179" xml:id="emml179">EMML 179</witness>
+               <witness corresp="EMML201" xml:id="emml201">EMML 201</witness>
+               <witness corresp="EMML629" xml:id="emml629">EMML 629</witness>
+               <witness corresp="EMML1279" xml:id="emml1279">EMML 1279</witness>
+               <witness corresp="EMML1531" xml:id="emml1531">EMML 1531</witness>
+               <witness corresp="EMML1768" xml:id="emml1768">EMML 1768</witness>
+               <witness corresp="EMML1950" xml:id="emml1950">EMML 1950</witness>
+               <witness corresp="EMML2080" xml:id="emml2080">EMML 2080</witness>
+               <witness corresp="EMML2440" xml:id="emml2440">EMML 2440</witness>
+               <witness corresp="EMML3470" xml:id="emml3470">EMML 3470</witness>
+               <witness corresp="EMML4437" xml:id="emml4437">EMML 4437</witness>
+               <witness corresp="EMML6281" xml:id="emml6281">EMML 6281</witness>
+               <witness corresp="EMML6974" xml:id="emml6974">EMML 6974</witness>
+               <!--  the following mss are said to be used by Nickelsburg 2001, but one needs to check more to understand. For the moment I left him out.
+               <witness corresp="EMML6686" xml:id="emml6686">EMML 6686</witness>
```

Comment:
As we have discussed, the record ideally should list ALL KNOWN TESTIMONIES OF A TEXT, independently of whether or not they have been used in an edition or translation.

So not only these should be listed but also the other mss discovered by Ted, that is EMML 7942, 8098, 8234, 8347, 8400, 8433, 8544, 8546, 8548, 8560, 8608, 8703, 8768, 8775, 8846, 8943, 9009, 
Jerusalem EOTC 5e (stub probably needed)
Tel Aviv - TAfait005 (already present)
Parma 3843 (stub needed, not yet at https://www.ipocan.it/index.php/it/ms either)
GG00151 (already present)
NLA3 (already present)
NLA23 (already present)
NLA24 (already present)
DimmaUnesco1004 (already present)
DimmaUnesco1043 (already present)
DW01 (already present)
BDLaethd18 (already present)
Schoyen 2657 (to create)
Remnant (to create)
IES 392, 1736
EMIP00746
EMIP00937
EMIP00949
EMDA106 (Mota)
EMDA202 (Qaranyo)
EStny008

Reply:
(since this is going to be a model record to be referred during the exhibition it would be good to have everything in place)

Reply:
@nafisa-valieva if you think that inserting pointers in all relevant records is feasible - try it out now here, and instead of putting the list of witnesses, encode all the `msItems `in the `msContents `of the relevant manuscripts. (note that in most cases the Book of Enoch is by far not the only ID to point to, these are multiple text manuscripts.) once you have a clear idea of the time investment necessary for that maybe you will indeed demonstrate that "adding this tiny bit of information is not very time-consuming". would you like to try? https://github.com/BetaMasaheft/Documentation/issues/2699

Reply:
in this case the manuscripts are described in the article, and many of them have stub records already, and most of the EMML ones are already catalogued on vHMML. since this is a model record you surely can try your suggested approach in this case. 
yet note that still `rend="edition"` should be used for manuscripts used for the edition encoded in the Bm record itself, with the sigla used in the Bm apparatus, not any possible edition around there.